src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -43,10 +43,21 @@ import {
   formatInteger,
 } from 'components/Charts/utils';
 
+type County = {
+  county: string;
+  county_url_name: string;
+  county_fips_code: string;
+  state_fips_code: string;
+  state_code: string;
+  full_fips_code: string;
+  cities: string[];
+  population: string;
+};
```

Comment:
Thank you for this! 😁  I'm not sure where it should go either.  I feel like we probably need a util/counties.ts file with this type and some helpers for dealing with the county dataset.  Maybe add a TODO, and I might try to grab it as tech debt later this week.

Reply:
Added a TODO!